recipes/caffe2/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.8.dev" %}
+
+package:
+  name: caffe2
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/caffe2/caffe2.git
```

Comment:
[Build from tarballs, not repos.](https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos) We should probably package the 0.8.1 release instead.